app/test/Makefile

```diff
@@ -677,5 +671,4 @@ test-sheet-8: ${BUILD_DIR}/bin/zsv_sheet${EXE}
 	sleep 0.5 && \
 	tmux capture-pane -t $@ -p ${REDIRECT1} ${TMP_DIR}/$@.out && \
 	tmux send-keys -t $@ "q" && \
-	${CMP} ${TMP_DIR}/$@.out expected/$@.out && ${TEST_PASS} || ${TEST_FAIL})
-
+	${CMP} ${TMP_DIR}/$@.out expected/$@.out && ${TEST_PASS} || (echo 'Incorrect output:' && cat ${TMP_DIR}/$@.out && ${TEST_FAIL}))
```

Comment:
The `diff` command may be used instead of `cmp`.
In case of any differences, `diff` will dump the diff of files with non-zero exit status.